Fastfile

```diff
@@ -150,7 +150,12 @@ def _build_with_gym
   export_method = _get_export_method
   xcconfig_filename = Dir.pwd + '/TAB.release.xcconfig'
   create_xcconfig(filename: xcconfig_filename)
-  gym(export_method: export_method, xcconfig: xcconfig_filename)
+  gym(configuration: _configuration, export_method: export_method, xcconfig: xcconfig_filename)
+end
+
+def _configuration
+  value = get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: 'configuration')
```

Comment:
The `GYM_EXPORT_OPTIONS` are unrelated to the debug/release build configuration we're looking to add here right?